app.ts

```diff
@@ -42,10 +42,12 @@ app.post('/interactions', (req: Request<Record<string, unknown>, Record<string,
    * See https://discord.com/developers/docs/interactions/application-commands#slash-commands
    */
   if (type === InteractionType.APPLICATION_COMMAND) {
-    const { name } = data;
+    const command = data.command;
+    const opponent = data.opponent;
+    const wager = data.wager;

```

Comment:
I believe these will come from the `options` field of the request. ref - https://discord.com/developers/docs/interactions/application-commands .